boards/nucleo_f429zi/src/main.rs

```diff
@@ -44,7 +50,7 @@ const FAULT_RESPONSE: kernel::process::PanicFaultPolicy = kernel::process::Panic
 /// Dummy buffer that causes the linker to reserve enough space for the stack.
 #[no_mangle]
 #[link_section = ".stack_buffer"]
-pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
+pub static mut STACK_MEMORY: [u8; 0x4000] = [0; 0x4000];
```

Comment:
This is expected because of my terribly inefficient `tap_driver` implementation on the `tock-ethernet` branch. This PR is targeting this branch, and so we should focus on the diff concerning the added driver here. When it comes time to merge `tock-ethernet` into master, I think it's a good opportunity to consider the remaining infrastructure on that branch. Things are still WIP there.